cypress.json

```diff
@@ -9,5 +9,12 @@
     "reportTitle": "Uber Test Run",
     "overWrite": true,
     "saveHtml": true
+  },
+  "env": {
+    "nordmartWebUrl": "http://localhost:4200/",
+    "username": "moise.andrianekena@gmail.com",
+    "password": "moise",
+    "keycloakUrl": "http://192.168.99.100:8180"
```

Comment:
Avoid hard-coding, all of these should be consumed from environment variables.

Reply:
Something like this might do, or you can come up with a better solution. We will almost always have dynamic nordmartWebUrl, keycloakUrl

https://github.com/stakater-lab/stakater-nordmart-e2e-tests/blob/ed7f45f5563a2e4bce9f86d1d028bb9d92602861/cypress/integration/e2e/ProductsSearch.ts

Line 3 `cy.visit(Cypress.env('nordmartWebUrl'))`

Reply:
It can be seen as the default values and could be overwritten by --env argument when running cypress.